### PR TITLE
Chapter 14: Following users

### PR DESCRIPTION
**Pull request by Shyn1997**
screenshot from 2018-12-06 17-21-41

### CODE REVIEW

**Comment by Shyn1997** [comment]
ready

File: app/views/users/_unfollow.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<%= form_for(current_user.active_relationships.find_by(followed_id: @user.id),
```
Comment (DuongNT1512) [inline app/views/users/_unfollow.html.erb]: tuowng tự

**Comment by DuongNT1512** [comment]
commented

**Comment by Shyn1997** [comment]
ready

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class UsersController < ApplicationController
   before_action :load_gender, only: %i(new create)
   before_action :find_user, only: %i(show edit update correct_user destroy)
-  before_action :logged_in_user, only: %i(index destroy)
+  before_action :logged_in_user, only: %i(index destroy following followers)
```
Comment (DuongNT1512) [inline app/controllers/users_controller.rb]: ` following followers` ở đâu ra vậy?

**Comment by DuongNT1512** [comment]
commented

**Comment by Shyn1997** [comment]
ready

File: app/views/users/_unfollow.html.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<%= form_for @unfollow, html: { method: :delete }, remote: true do |f| %>
```
Comment (baclv) [inline app/views/users/_unfollow.html.erb]:
```suggestion
<%= form_for @unfollow, html: {method: :delete}, remote: true do |f| %>
```

**Pull request merged**